### ISSUE TITLE
Bug: I noticed that Obsidian was freezing on the trait text.

### ISSUE THREAD

**Issue by wangyu92** [platers/obsidian-linter#927]
- [ X ] I have verified that I am on the latest version of the Linter

## Describe the Bug

I was using Obsidian and noticed that my app kept freezing. I started by disabling all plugins and found that the freezing was caused by the Linter plugin. I also found that this happened all the time under certain circumstances.

## How to Reproduce

<img width="522" alt="image" src="https://github.com/platers/obsidian-linter/assets/20921042/ed7dd3a0-3d3b-4d12-a8eb-b63d7a88bbe5">

Example to reproduce issue with

``` markdown
---
tags: []
title: Bug Reproduce
date created: 2023-10-28T02:29:30
date modified: 2023-10-28T02:30:36
---
- Hi. This is a note to reproduce the bug.
- There must be at least one bullet point.

- Add an empty line and write one or more bullet points again like this.
- Then run the linter plugin and the Obsidian app should freeze.
```

## Expected Behavior

In short, your Obsidian app will freeze.

## Device

- [x] Desktop: Silicon mac
- [ ] Mobile (I haven't tried it on mobile.)

**Comment by pjkaufman** [comment]
Could you prived your `data.json`, @wangyu92 ? I know there are some situations that the Linter does not handle the best.

**Comment by pjkaufman** [comment]
Hey @wangyu92 , could you provide your `data.json`? Without it, I may not be able to address this potential issue.

**Comment by pjkaufman** [comment]
I am guessing this is caused by the Linter not using workers to Lint files. However without your `data.json` @wangyu92 , I will not be able to look into exactly what is causing the problem since I cannot reproduce locally with just the info provided.

**Comment by wangyu92** [comment]
I really sorry for too late reply. I upload `data.json` file.

[data.json](https://github.com/platers/obsidian-linter/files/13331794/data.json)

**Comment by pjkaufman** [comment]
Sorry about the delay in responding. Could you try out the following `main.zip` by unzipping its contents and copying the `main.js` file to `<vault_base>/.obsidian/plugins/obsidian-linter`? This could have been related to the Live Preview issues since I see that you are using live preview.

Here is the `main.zip`: 
[main.zip](https://github.com/platers/obsidian-linter/files/13648953/main.zip)

If you are not able to test it manually, the change for Live Preview should go out with the next release.

**Comment by pjkaufman** [comment]
Hey @wangyu92 , does this issue still exist for you? I see that the issue is pretty old and has not been updated in a while. I plan on closing this if I do not hear back since I cannot reproduce the issue in question.

**Comment by pjkaufman** [comment]
I have not seen anything new around this, so I am going to close th e issue. If the issue still exists, feel free to let me know.
